content/pages/homepage.html

```diff
@@ -154,97 +158,200 @@ <h3 class="ui large monospace header">
       </section>
     {% endblock homepage_feature_detail %}
 
-    <div class="ui divider"></div>
-
-    {% block homepage_bottom_callout %}
-      <section>
-        <div class="ui very padded container">
-          <div class="ui grid center aligned">
-
-            <div class="row">
-              <div class="column twelve wide computer sixteen wide tablet">
-
-                {#
-                  TODO replace this block with something more actionable. It's
-                  not saying anything new at this point in the page, and the call
-                  to action is not strong.
-                #}
-                <div class="ui basic vertical huge segment">
-                  <p>
-                    Whenever you push code,
-                    we automatically build and host your documentation.
-
-                    <b>Get your docs online in 5 minutes.</b>
-                  </p>
-
-                  <p>
-                    <a class="ui large teal stackable button"
-                    data-analytics="signup-modal"
-                     onclick="jQuery('#homepage-modal').modal('show');">
-                    <i class="fad fa-play icon"></i>
-                    Get started
-                  </a>
-                  </p>
-                </div>
-
+<div class="ui container">
+      <div class="ui horizontal divider">Try it out</div>
+    </div>
+    
+    <section id="try-it-out">
+      <div class="ui padded container">
+        <h2 class="ui huge center aligned header">
+          Upgrade your documentation with Read the Docs
+        </h2>
+    
+        <div class="ui very padded centered grid">
+    
+          <div class="ten wide computer sixteen wide tablet sixteen wide mobile column">
+            <div class="ui large left aligned basic vertical segment">
+    
+              {# Stylized code blocks #}
+              {% call technical.code_block(title="<code>.readthedocs.yaml</code>", icon="fa-file") %}
+              {% markdown %}
+    
+              ```yaml
+              version: 2
+              build:
+                os: ubuntu-22.04
+                tools:
+                  python: "3.10"
+                  # You can also specify other tool versions:
+                  # nodejs: "16"
+    
+              # Build documentation in the docs/ directory with Sphinx
+              sphinx:
+                 configuration: docs/conf.py
+    
+              # Dependencies required to build your docs
+              python:
+                 install:
+                 - requirements: docs/requirements.txt
```

Comment:
Great call, this is a great block for the front page. This would be another place to make these working examples instead, right now this is vaguely correct but won't work if the user tries to do anything with it.

Reply:
Agreed -- would be neat to have a "deploy this example" button, and they just need to login with GH, and it magically deploys docs. We already have the example projects, so it wouldn't be hard to configure this.

Reply:
For what it's worth, it's basically the exact config we have in our example Sphinx project:

https://github.com/readthedocs-examples/example-sphinx-basic/

I thought our onboarding prompts users to install this right, so it should mostly work?

Reply:
A one click deploy option would be really neat. I don't know what we have configured for that sort of workflow currently, though know the tutorial at least promotes the template repository at least.

I was suggesting this all as a next step, so nothing block here of course. These would be good options to explore for later though